wellcad/com/_borehole.py

```diff
@@ -21,7 +21,7 @@ class Borehole(DispatchWrapper):
                          "CalculateApparentMetalLoss", "GetLog", "CreateNewWorkspace",  "Workspace", "FileExport",
                          "ConvertLogTo", "FilterLog", "ResampleLog", "InterpolateLog", "ElogCorrection",
                          "NMRFluidVolumes", "ROPAverage", "SharpenRGBLog", "RetinexFilterRGBLog", 
-                         "Transmissivity", "ShearWaveVelocity", "EllipseFitting", "BreakoutAutoPick", "HorzStress")
+                         "Transmissivity", "ShearWaveVelocity", "EllipseFitting", "BreakoutAutoPick", "HorzStress", "DuplicateLog")
```

Comment:
Can you please confirm you actually need to add this here? In general the autoregistration should work.

Reply:
> Can you please confirm you actually need to add this here? In general the autoregistration should work.

Yes, when testing whithout it I have this error message 

Error
Traceback (most recent call last):
  File "C:\Projects\pywellcad\test\test_borehole.py", line 326, in test_duplicate_log
    duplicate_log_litho1 = self.borehole.duplicate_log(str(log_litho._dispatch))
  File "C:\Projects\pywellcad\wellcad\com\_borehole.py", line 4421, in duplicate_log
    return Log(self._dispatch.DuplicateLog(log))
TypeError: 'NoneType' object is not callable